Buildings/ThermalZones/Detailed/Validation/BESTEST/Cases6xx/Case600.mo

```diff
@@ -73,6 +73,47 @@ model Case600 "Case 600FF, but with dual-setpoint for heating and cooling"
   Buildings.Controls.OBC.CDL.Reals.MovingAverage PCoo(delta=3600)
     "Hourly averaged cooling power"
     annotation (Placement(transformation(extent={{-20,-8},{-12,0}})));
+  Modelica.Blocks.Sources.RealExpression hGloHor(
```

Comment:
Always add the unit to these expressions.

Reply:
added.